docs/use-cases/tokenization/real-world-assets.page.tsx

```diff
@@ -0,0 +1,431 @@
+  },
+  {
+    title: "Purpose-Built for Finance",
+    description:
+      "XRP Ledger provides out-of-the-box institutional-grade functionality, reducing development overhead and eliminating the need for smart contracts.",
+    number: "02",
+  },
+  {
+    title: "Native Compliance & Security",
+    description:
+      "Maintain control over tokenized assets and user access with XRP Ledger's asset control features, multi-signature accounts, and built-in compliance tools.",
+    number: "03",
+  },
+  {
+    title: "Pathfinding & Auto-Bridging for Liquidity",
+    description:
+      "Streamline cross-currency transactions and trading with XRP Ledger's auto-bridging and pathfinding features to automatically increase liquidity between issued tokens and XRP.",
+    number: "04",
+  },
+];
+
+function FeatureCard({ title, description }) {
+  const { useTranslate } = useThemeHooks();
+  const { translate } = useTranslate();
+  return (
+    <article className="feature-card">
+      <header className="feature-header">
+        <h2 className="feature-title">{title}</h2>
+      </header>
+      <p className="feature-description">{translate(description)}</p>
+    </article>
+  );
+}
+
+function RwaTokenizationFeatures() {
+  const { useTranslate } = useThemeHooks();
+  const { translate } = useTranslate();
+  return (
+    <section className="rwa-tokenization">
+      <div className="container max-w-1150">
+        <header className="rwa-header">
+          <h2 className="rwa-title">
+            {translate("Why Developers Choose XRPL for RWA Tokenization")}
+          </h2>
+          <p className="rwa-subtitle">
+            {translate(
+              "A developer-first blockchain for building efficient, scalable tokenization solutions."
+            )}
+          </p>
+        </header>
+        <div className="feature-grid">
+          {featuresToken.map((feature, index) => (
+            <FeatureCard key={index} {...feature} />
+          ))}
+        </div>
+        <div className="cta-container">
+          <Link
+            to="/docs/concepts/tokens/decentralized-exchange/autobridging"
+            className="btn btn-primary small-100 auto-bridge"
+          >
+            {translate("Learn About Auto-bridging")}
+          </Link>
+          <Link
+            to="/docs/concepts/tokens/fungible-tokens/paths"
+            className="btn btn-link"
+          >
+            {translate("Explore Pathfinding")}
+          </Link>
+        </div>
+      </div>
+    </section>
+  );
+}
+
+function FeatureItem({ title, link }) {
+  const { useTranslate } = useThemeHooks();
+  const { translate } = useTranslate();
+  return (
+    <li onClick={() => window.open(link, "_blank")} className="feature-item">
+      <div className="feature-item__content">
+        <span className="feature-item__title">{translate(title)}</span>
+        <span className="right-arrow-item"> </span>
+      </div>
+      <div className="feature-item__divider"></div>
+    </li>
```

Comment:
It's bad practice to use `onclick` when all you want is something that works like a link—it breaks behavior in subtle ways, like middle-clicking won't open it in a new tab and SEO doesn't catch things.

But furthermore, this whole component is re-creating one we already have defined. I've adjusted the shared component and refactored this page to use it here: 6835e174b42f5d4c84ea810670f0185d670b4c5e . Feel free to pull that commit and use it.